server/src/testFixtures/java/io/crate/expression/scalar/ScalarTestCase.java

```diff
@@ -282,6 +277,27 @@ public void assertCompile(String functionExpression,
         assertThat(compiled).satisfies(matcher.apply(scalar));
     }
 
+    @SuppressWarnings({"rawtypes", "unchecked"})
+    protected Scalar compileScalar(String functionExpression, String user, UserLookup userLookup) {
+        Function function = buildFunctionSymbol(functionExpression);
+        Scalar scalar = buildScalar(function);
+        return scalar.compile(function.arguments(), user, userLookup);
+    }
+
+    private Function buildFunctionSymbol(String functionExpression) {
+        Symbol functionSymbol = sqlExpressions.asSymbol(functionExpression);
+        functionSymbol = sqlExpressions.normalize(functionSymbol);
+        assertThat(functionSymbol)
+            .as("function expression was normalized, compile would not be hit")
+            .isNotInstanceOf(Literal.class);
+        return (Function) functionSymbol;
+    }
+
+    @SuppressWarnings("rawtypes")
+    private Scalar buildScalar(Function function) {
+        return (Scalar) sqlExpressions.nodeCtx.functions().getQualified(function);
+    }
```

Comment:
Is this really worth a dedicated function? no strong opinion but for me this doesn't add value but just adds indirection I'd have to look up.